kafka-rest/pom.xml

```diff
@@ -59,6 +59,12 @@
             <artifactId>slf4j-log4j12</artifactId>
             <version>${log4j.version}</version>
         </dependency>
+        <!-- Use a repackaged version of log4j with security patches. Default log4j v1.2 is a transitive dependency of slf4j-log4j12, but it is excluded in common/pom.xml -->
+        <dependency>
+            <groupId>io.confluent</groupId>
+            <artifactId>confluent-log4j</artifactId>
+            <version>${confluent-log4j.version}</version>
```

Comment:
Is this needed?

Reply:
Do you mean the version line? I don't think it's strictly needed, but just following the convention in the rest of the `pom.xml` to include the version.